src/ui/server/controllers/insurance/policy/save-data/loss-payee-financial-details-uk/index.ts

```diff
@@ -0,0 +1,46 @@
+import api from '../../../../../api';
+import getDataToSave from '../../../../../helpers/get-data-to-save';
+import { sanitiseData } from '../../../../../helpers/sanitise-data';
+import stripEmptyFormFields from '../../../../../helpers/strip-empty-form-fields';
+import { Application, RequestBody } from '../../../../../../types';
+
+import POLICY_FIELD_IDS from '../../../../../constants/field-ids/insurance/policy';
+
+const { FINANCIAL_ADDRESS } = POLICY_FIELD_IDS;
+
+/**
+ * string fields which are exempt from being stripped by stripEmptyFormFields
+ * for example when a string field needs to be set to an empty string or null
+ */
+export const NULL_OR_EMPTY_STRING_FIELDS = [FINANCIAL_ADDRESS];
+
+/**
+ * lossPayeeFinancialDetailsUk
+ * Strip invalid fields from submitted form data and update the application.
+ * This is used for any save functionality in the Policy - Loss payee financial details Uk part of the application.
+ * @param {Application} Application
+ * @param {Express.Request.body} Form data
+ * @param {Object} Field error list
+ * @returns {Object} Saved data
```

Comment:
Returns `Promise<Object>`

Reply:
Have noticed in this PR few have been updated already, perhaps update all files in this PR and then a separate PR for rest?